### DISCUSSION TITLE
Does Concourse UI support enduser's confirmation popup when triggering a job manually from UI.

### DISCUSSION THREAD

**Discussion by wayoubi** [concourse/concourse#8905] [Ideas]
Hello,  Across me work I might need sometimes to pause/ trigger concourse jobs from the UI. I have always found it a best practice to ask the end user to confirm his action when triggering a process manually specially when this process change system state.  

I searched the internet and found that In the context of ITIL, ensuring proper confirmation for actions that may change state is crucial. When a user initiates an action from the UI that could alter the system’s state, a popup message can serve as an effective mechanism to confirm their intent and prevent accidental or unintended changes. In the ITIL framework, specific guidelines regarding popup confirmation messages are not explicitly outlined. However, the concept aligns with best practices for user experience and change management. 

I was wondering if this is something we can enable in concourse by configuration or may to adopt it as a feature in future releases.

**Comment by taylorsilva** [comment]
There is no such feature in Concourse right now. You can implement your own kinda of "manual approval" within your tasks. I've done it for `terraform apply` tasks that I want to double check before running. Wrote up how that works here: https://www.taylorsilva.com/posts/concourse-manual-approval-step/
